src/elexmodel/handlers/s3.py

```diff
@@ -128,7 +132,7 @@ def wait_for_versions(self, q):
             try:
                 future.result()
                 yield version, data
-            except Exception as e:
+            except Exception as e:  # pylint: disable=broad-exception-caught
```

Comment:
this is just to remove the tox error? We haven't really cared about those in the past, since we use our pre-commit hooks to make sure we are pep compliant.

Reply:
In that case maybe we should disable `pylint` altogether.  If we're just going to ignore its complaints, we might as well speed up our `tox` runs 😄 

Another option here would be to add more things to our list of `pylint` complaints we ignore in `setup.cfg`, although I'm hesitant to do that with too many things because a lot of these are useful, but there are cases where we want to ignore them.  This one in particular is here because `boto3` and its related libraries can raise several different types of `Exception`s and we want to handle them all the same way.  Ideally, we'd break that out or specify the types but honestly, I find it hard to know up front which exception types `boto3` can throw 😬

Reply:
I think the answer is we should stop using tox and use pytest directly in the unit tests. Since there is no need to use two different linters

Reply:
Hmmmmmmm...I'd argue in favor of using `tox` but removing `pylint` from it.  `tox` is great because we can specify multiple Python versions and it'll run whatever suite of tests etc. for each version in isolated environments. We can also have it run `pre-commit` for us, e.g. https://github.com/jugmac00/flask-reuploaded/blob/master/tox.ini 🤔   But yeah it looks pretty easy to remove `pylint` from `tox`, which is good 😄

Reply:
@lennybronner alright so I did some reading here and it seems like `pylint` is designed with the intention of being overly-cautious.  The best way I saw it described is that `pylint` is a superset of `flake8`.  We probably don't need the extra complaints `pylint` generates on top of `flake8` 🤔 

So yeah, let me know if you want me to remove `pylint` in this PR or open another one 😄